src/DssSpell.sol

```diff
@@ -19,6 +19,12 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
+interface VatLike {
+    function Line() external view returns (uint256);
+    function file(bytes32 ilk, uint256 data) external;
```

Comment:
Interface is wrong. We nitted this earlier this spell.

Try running `cast interface 0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B` to fix this.

```suggestion
    function file(bytes32 what, uint256 data) external;
```